03/app.js

```diff
@@ -1,2 +1,8 @@
-const n = 24;
-const oddNumbers = [];
\ No newline at end of file
+const n = 100;
+const oddNumbers = [];
+
+for (let i = 0;i < n; i += 2 ) {
+    oddNumbers.push(i + 1);
```

Comment:
Skoro tak, to wolałbym zacząć od 1 i nie dodawać jej przy `.push()` tj. `let i = 1` :P